src/components/AddressResolver/EndpointEntry/EndpointEntry.test.tsx

```diff
@@ -1,9 +1,11 @@
 import React, { FunctionComponent } from "react";
 import { fireEvent, render, screen, waitFor } from "@testing-library/react";
 import { EndpointEntry } from "./EndpointEntry";
-import { getFeatures } from "../../../services/addressResolver";
+import { getFeatures } from "@govtechsg/address-identity-resolver";
 
-jest.mock("../../../services/addressResolver");
+jest.mock("@govtechsg/address-identity-resolver", () => ({
```

Comment:
I think function after `@govtechsg/address-identity-resolver` is not needed as we are mocking the item in line 11

Reply:
ok